tests/CronDate.test.ts

```diff
@@ -0,0 +1,381 @@
+import { CronDate } from '../src/index.js';
+import { TimeUnits } from '../src/types.js';
+
+describe('CronDate tests', () => {
+  test('is the last weekday of the month', () => {
+    // Last monday of september
+    const date1 = new CronDate(new Date(2021, 8, 27));
+    expect(date1.isLastWeekdayOfMonth()).toBe(true);
+
+    // Second-to-last monday of september
+    const date2 = new CronDate(new Date(2021, 8, 20));
+    expect(date2.isLastWeekdayOfMonth()).toBe(false);
+  });
+
+  test('CronDate should handle addSecond correctly', () => {
```

Comment:
```suggestion
  test('should handle addSecond correctly', () => {
```

We probably won't need prefixes for test cases.